tests/test_fuzzbucket.py

```diff
@@ -673,6 +673,83 @@ def fake_describe_images(ImageIds=(), *_, **__):
         assert response.json["boxes"] != []
 
 
+@pytest.mark.parametrize(
+    ("authd", "update_body", "expected"),
+    [
+        pytest.param(
+            True,
+            {
+                "instance_tags": {"withered": "hand", "early": "seasons"},
+                "ttl": "108000.0",
```

Comment:
It might be good to enumerate a couple more combinations, e.g. tags without TTL and TTL without tags.